.github/workflows/build-release-artifacts.yml

```diff
@@ -0,0 +1,25 @@
+name: Build Release Artifacts
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - 'releases/**'
+
+jobs:
+  create-branches:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Perform gradle build
+        run: |
+          ./gradlew  firebasePublish -PprojectsToPublish=firebase-firestore -PpublishMode=RELEASE -PincludeFireEscapeArtifacts=true
```

Comment:
I think hardcoding is good in this initial version. Eventually we want to make the value of `-PprojectsToPublish` coming from `release.cfg`?

What I'm thinking -- Maybe we can make a [custom action](https://docs.github.com/en/actions/creating-actions/about-custom-actions) around `release.cfg` (for its creation, parsing, etc). That way it is possible to be used in multiple workflows and its input/output format can be defined based on its intended usage.

Although this is just one way to design it.